cmake/msys/msys_helper.sh

```diff
@@ -90,33 +90,33 @@ if [ $? -ne 0 ]; then
 	if [ $? -ne 0 ]; then
         	err "ERROR: Could not build/install fltk -- Zyn needs this.  Exiting."
 	fi
-	
-#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe 	
+
+#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe
 	popd
 else
-	warn "  - Skipping, fluid binary already exists" 
+	warn "  - Skipping, fluid binary already exists"
 fi
 
 info "Downloading and building libogg $oggver"
 
-if [ ! -e $mingw_root/lib/libogg.dll.a ]; then 
-	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O $HOME/libogg-source.tar.xz
+if [ ! -e "$mingw_root/lib/libogg.dll.a" ]; then
+	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O "$HOME/libogg-source.tar.xz"
```

Comment:
> But I don't see why we should quote this as it's currently not needed

Because people will continue to copy and paste it (as they have previously), and we may calculate the values.

> But I don't see why we should quote this as it's currently not needed, probably won't (space in a version number in an url is not a common thing)

And `$HOME` is?  I've used a lot of systems and I can promise you `%20` in a URL is more common than `<space>` in a `/home` path.

> makes the code harder to read

@zapashcanon arguably... all this PR does is make the code harder to read, that portion is quite irrelevant.

Adding double quotes to the URL does a two things in my eyes.

1. Helps make the script consistent.  For those that don't have the accuracy of shellcheck, it's a good habit!
2. Helps copy/paste, especially if the version becomes parametrized or calculated down the road.  This is very likely for scripts like this that are growing rather rapidly, link against specific versions and have a lot of redundant code.

That said, it's fine to merge without.  I won't hold up the patch for this alone.

>  plan to add shellcheck with astyle and kwstyle

We should add it separately.  We only have a handful of scripts, and this is one more greenlight we can have for the PR process.

Thanks for your work on this!